### ISSUE TITLE
Feature Request: Give an option to make the window title applet a clickable button

### ISSUE THREAD

**Issue by keyb0ardninja** [psifidotos/applet-window-title#121]
Give a toggle to make the window title applet a clickable button which will open a combination of the Alt+F3 menu and the menu which opens when right clicked on that entry in Latte Dock.

For example in case of Firefox, the Alt+F3 menu looks like this:

![image](https://user-images.githubusercontent.com/16205994/137576619-2b628540-f2fc-403c-b731-d03f527eaf5e.png)

and the right-click on latte menu looks like this:

![image](https://user-images.githubusercontent.com/16205994/137576635-ea24e251-9302-417c-a294-1dfca86bcd44.png)

I feel that these menus contain useful actions which a user would find beneficial if given access through clicking on the window title applet. Mainly because someone who uses the window title applet probably has hidden the titlebar (like me) and the Alt+F3 menu can now only be accessed by keyboard shortcut. I've also set Latte Dock to auto-hide whenever a window goes over it. So, most of the time, the dock is not directly shown on my desktop, but the window title applet is shown 100% of the time.

By the way, the application title on the macOS global menu is also a clickable menu.

**Comment by keyb0ardninja** [comment]
Thanks for the quick response and also for the bug report! I'll keep an eye on the bug report. Can you try to implement the menu from the latte-dock-right-click-menu-on-entry for now so that the groundwork will be ready whenever (or if) the Alt+F3 menu is made available from upstream?

Thanks again for the applets and Latte Dock. They're part of my day to day workflow and they work extremely well :)

**Comment by psifidotos** [comment]
makes sense and this is why I leave it open. But it would increase code complexity a lot and needs many adjustments from the Latte Tasks paradigm, so we will see, currently I dont have any interest about it.

**Comment by keyb0ardninja** [comment]
By the way, I actually came here to open this feature request when I was reading Nate's [blog](https://pointieststick.com/2021/10/15/this-week-in-kde-plasma-25th-anniversary-edition-is-released/) and saw this screenshot:

![image](https://user-images.githubusercontent.com/16205994/137577694-3691023e-9544-4e96-86c6-b94175be9ca8.png)

This is exactly the menu I want. It seems like the default plasma task manager already has what I want: the combination of the Alt+F3 menu and the right click context menu of the entry. But the task manager is also a plasma applet, right? Can you take a look into how the default plasma task manager is doing it?
